Source/Plugins/Core/com.equella.core/js/__stories__/search/OwnerSelector.stories.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * The Apereo Foundation licenses this file to you under the Apache License,
+ * Version 2.0, (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from "react";
+import OwnerSelector from "../../tsrc/search/components/OwnerSelector";
+import { action } from "@storybook/addon-actions";
```

Comment:
Stylistically, I've personally found it easier to have node module import first, the internal imports after.

```suggestion
import { action } from "@storybook/addon-actions";
import OwnerSelector from "../../tsrc/search/components/OwnerSelector";
```

Reply:
Yes, I was wondering about this. We've accrued a bit of a mess. Is there an ESLint rule you've ever found to support this notion?

I'd be interested (in another place) go through and tidy this all up. (I'm disappointed that the intellij optimise imports does not do something like this - than again, maybe there's an option...)

Reply:
eslint and eslint-plugin-import come with https://eslint.org/docs/rules/sort-imports and https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md which can be setup.
Alternatively there is https://github.com/lydell/eslint-plugin-simple-import-sort

Reply:
Nice. The simple-import-sort looks nice and... simple. :grin: